src/Config/CliBuilder.ts

```diff
@@ -64,13 +64,23 @@ export class CliBuilder {
       })
       .option('proxy', {
         requiresArg: true,
+        deprecated: true,
```

Comment:
Is it correct? I have not found any information that this param should be deprecated.

Reply:
Yes, according to the task, I think so.
But I asked Art.

Reply:
It was wrong. I'll fix it.